### PR TITLE
Add telephone number as 'first class' field in Application Summary

### PR DESCRIPTION
**Pull request by Harriethw**
# Context

We want to pull out the contact number entered within the application form as a 'first class' field, in order to highlight it to assessors.
API work to be merged in once this is complete

https://github.com/ministryofjustice/hmpps-approved-premises-api/pull/1328 

# Changes in this PR

* sending the field as a param when submitting the application
* using shared Application Summary component to add the contact number in
* I've also added [the application ID as it was needed for assessors] as I was in the component already (https://trello.com/c/1l6BNsLO/772-add-application-id-to-assessor-view) 

## Screenshots of 
CYA with added ID
CYA_view

Assessor submitted application with added contact number
assessor_view

Referrer submitted application view with added contact number
referrer_view

# Release checklist

As part of our continuous deployment strategy we must ensure that this work is
ready to be released at any point. Before merging to `main` we must first
confirm:

## Pre merge checklist

- [ ] Are any changes required to the e2e tests?
- [ ] If you've added a new route, have you added a new
  `auditEvent`? (see `server/routes/apply.ts` for examples)
- [ ] Are there environment variables or other infrastructure configuration which needs to be included in this release?
- [ ] Are there any data migrations required. Automatic or manual?
- [ ] Does this rely on changes being deployed to the CAS API?

## Post merge checklist

Once we've merged it will be auto-deployed to the dev environment.

[Find the build-and-deploy job in CircleCI](https://app.circleci.com/pipelines/github/ministryofjustice/hmpps-community-accommodation-tier-2-ui).

Should a release fail at any step, you as the author should now lead the work to
fix it as soon as possible.

### CODE REVIEW

File: server/utils/applications/managementInfoFromAppData.test.ts [hunk 1]
```diff
@@ -88,4 +89,35 @@ describe('managementInfoFromAppData', () => {
...
+        'referrer-details': null,
+      },
+      {
+        'referrer-details': { 'contact-number': null },
+      },
+      {},
+      null,
+    ]
+
+    it.each(noDateData)('returns null if no contact number is given', data => {
```
Comment (libuk) [inline server/utils/applications/managementInfoFromAppData.test.ts:149]: Thorough test. Nice!

File: server/utils/applications/managementInfoFromAppData.ts [hunk 2]
```diff
@@ -28,4 +28,17 @@ const conditionalReleaseDateFromAppData = (application: Application): string =>
   return date || null
 }
 
-export { preferredAreasFromAppData, hdcEligibilityDateFromAppData, conditionalReleaseDateFromAppData }
+const telephoneNumberFromAppData = (application: Application): string => {
```
Comment (libuk) [inline server/utils/applications/managementInfoFromAppData.ts:31]: I think the return value should be `string | null`? A bit surprised TS hasn't complained about that.

File: server/views/applications/show.njk [hunk 3]
```diff
@@ -24,24 +25,16 @@ printButton %}
...
-            <span class="govuk-!-font-weight-bold">Applicant name: </span>{{ application.person.name }}</li>
-          <li>
-            <span class="govuk-!-font-weight-bold">Prison number: </span>{{ application.person.nomsNumber }}</li>
-          <li>
-            <span class="govuk-!-font-weight-bold">Submission date:</span>
-            {{ formatDate(application.submittedAt, {format: 'medium'}) }}
-          </li>
-        </ul>
-      </div>
+      {{ applicationSummary({
```
Comment (libuk) [inline server/views/applications/show.njk:28]: 👍🏾

**Review by libuk — APPROVED** [review]
LGTM, just a small comment on types.

File: server/utils/applications/managementInfoFromAppData.test.ts (hunk 1 above)
Comment (libuk) [inline server/utils/applications/managementInfoFromAppData.test.ts:149]: Thorough test. Nice!

File: server/views/applications/show.njk (hunk 3 above)
Comment (libuk) [inline server/views/applications/show.njk:28]: 👍🏾

File: server/utils/applications/managementInfoFromAppData.ts (hunk 2 above)
Comment (Harriethw) [reply server/utils/applications/managementInfoFromAppData.ts:31]: hmm yeah good point

**Review by libuk — APPROVED** [review]

**Pull request merged**